server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -244,6 +244,9 @@ export function configureWebSocketServices(
 
 		let connectDocumentComplete: boolean = false;
 		let connectDocumentP: Promise<void> | undefined;
```

Comment:
Should we remove the `disconnect` handler's `await connectDocumentP` and instead fully rely on `connectDocument.finally()` to retry? Or is it OK to have both?

Perhaps we can have both for now, then experiment with just 1 if this solution solves the problem? We know that the `await connectDocumentP` solution _helps_ but is not complete, so I don't want to remove it prematurely.

Reply:
Agreed. Let's keep both.